src/counts.jl

```diff
@@ -9,7 +9,16 @@
 typealias IntUnitRange{T<:Integer} UnitRange{T}
 
 #### functions for counting a single list of integers (1D)
+"""
+    addcounts!(r::AbstractArray, x::IntegerArray, levels::IntUnitRange)
```

Comment:
`IntegerArray` and `IntUnitRange` are unexported type aliases. It would probably be better to show the actual types:
```julia
   addcounts!{T<:Integer, U<:Integer}(r::AbstractArray, x::AbstractArray{T}, levels::UnitRange{U}) 
```
or maybe (but this isn't correct Julia syntax):
```julia
   addcounts!(r::AbstractArray, x::AbstractArray{<:Integer}, levels::UnitRange{<:Integer}) 
```